templates/cli/README.md.twig

```diff
@@ -0,0 +1,30 @@
+# {{ spec.title }} {{sdk.name}} SDK
+
+![License](https://img.shields.io/github/license/{{ sdk.gitUserName|url_encode }}/{{ sdk.gitRepoName|url_encode }}.svg?v=1)
+![Version](https://img.shields.io/badge/api%20version-{{ spec.version|url_encode }}-blue.svg?v=1)
+{% if sdk.warning %}
+
+{{ sdk.warning }}
+{% endif %}
+
+{{ sdk.description }}
+
+{% if sdk.logo %}
+![{{ spec.title }}]({{ sdk.logo }})
+{% endif %}
+
+## Installation
```

Comment:
We should change installation instructions once we know how we want to install the alias on users OS.

Reply:
Done